build.gradle

```diff
@@ -88,6 +88,11 @@ subprojects { subproject ->
     }
 
 	configurations {
+		all {
+			resolutionStrategy.dependencySubstitution {
+				substitute module('org.hamcrest:hamcrest-core') using module('org.hamcrest:hamcrest:2.2')
+			}
+		}
```

Comment:
Can we add a comment on why we are doing this?

Reply:
Added an explanation using the `because` construct that is optionally allowed here.